sql/sql_statistics.h

```diff
@@ -144,6 +144,8 @@ double get_column_range_cardinality(Field *field,
                                     uint range_flag);
 bool is_stat_table(const LEX_CSTRING *db, LEX_CSTRING *table);
 bool is_eits_usable(Field* field);
+uint get_offset_to_value(Field *field);
+uchar *get_buffer_end(uchar *to);
 
```

Comment:
These are only used in sql_statistics.cc (once each) so should at least be made static.

Maybe they also should be named differently and/or moved somewhere because it looks to me like these functions assume some storage format but it is not immediately clear where that format is defined, etc.

Reply:
Good point. I usually clean these sorts of functions up, I missed this.